Parser.tests/Tests/MathsProcessorTests.ts

```diff
@@ -1,6 +1,11 @@
 ﻿/// <reference path="../references.ts" />
 
-"use strict";
```

Comment:
What about "use strict"?

Reply:
I'm 99% sure you don't need it anymore depending on what we're building for but will double check. I was gonna remove from all when time allows if I'm right